src/types.js

```diff
@@ -3,3 +3,54 @@ export type Data = {
   key: string,
   value: string,
 };
+
+export type GlobalAttributes = $Shape<{|
+  hidden: boolean,
```

Comment:
does every component accept these or what is this for exactly?

Reply:
Yes: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden.

> The `hidden` global attribute is a Boolean attribute indicating that the element is not yet, or is no longer, relevant.
> For example, it can be used to hide elements of the page that can't be used until the login process has been completed.